diff --git a/bolt/tools/merge-fdata/CMakeLists.txt b/bolt/tools/merge-fdata/CMakeLists.txt
index f6a87a226898..d20b578ed43d 100644
--- a/bolt/tools/merge-fdata/CMakeLists.txt
+++ b/bolt/tools/merge-fdata/CMakeLists.txt
@@ -12,6 +12,6 @@ add_bolt_tool(merge-fdata
 add_dependencies(bolt merge-fdata)
 
 # Emit relocations for BOLT meta test (bolt/test/runtime/meta-merge-fdata.test)
-if (BOLT_INCLUDE_TESTS AND UNIX AND NOT APPLE)
-  target_link_options(merge-fdata PRIVATE LINKER:--emit-relocs)
-endif()
+#if (BOLT_INCLUDE_TESTS AND UNIX AND NOT APPLE)
+#  target_link_options(merge-fdata PRIVATE LINKER:--emit-relocs)
+#endif()
